test/csuite/random_directio/main.c

```diff
@@ -134,20 +134,20 @@ static const char *const uri_rev = "table:rev";
  * create table:A101-3          (starts a new sequence)
  *
  * insert k/v 102 into table:main
- * rename table:A100-3 -> table:B100-3  (third step in sequence)
+ * update key in table:A100-3           (third step in sequence)
  * insert into table:A101-3             (second step in sequence)
  * create table:A102-3                  (starting new sequence)
  *
  * insert k/v 103 into table:main
- * update key in table:B100-3          (fourth step)
- * rename table:A101-3 -> table:B101-3 (third step)
+ * update key in table:B100-3
+ * update key in table:A101-3
  * insert into table:A102-3
  * create table:A103-3
  *
  * insert k/v 104 into table:main
- * drop table:B100-3                   (fifth and last step)
- * update key in table:B101-3          (fourth step)
- * rename table:A102-3 -> table:B102-3
+ * drop table:A100-3                   (fourth and last step)
+ * update key in table:B101-3
+ * update key in table:A102-3
  * insert into table:A103-3
  * create table:A104-3
  * ...
```

Comment:
I think this comment showing the example sequence is close, but not quite right (for one thing, there is no longer a table:B101-3.   Here's what I think the comment should be:
```
 *
 * insert k/v 99 into table:main      (with no additional schema operations)
 *
 * insert k/v 100 into table:main
 * create table:A100-3       (3 for thread #3)
 *
 * insert k/v 101 into table:main
 * insert into table:A100-3     (continuing the sequence)
 * create table:A101-3          (starts a new sequence)
 *
 * insert k/v 102 into table:main
 * update key in table:A100-3           (third step in sequence)
 * insert into table:A101-3             (second step in sequence)
 * create table:A102-3                  (starting new sequence)
 *
 * insert k/v 103 into table:main
 * drop table:A100-3                   (fourth and last step)
 * update key in table:A101-3
 * insert into table:A102-3
 * create table:A103-3
 * ...
```

Reply:
Thanks, this was super helpful! Updated here [af1c647](https://github.com/wiredtiger/wiredtiger/pull/10306/commits/af1c647aef15535e42f88519d5de76dc29ca1f68)